test/integration-serial/run_serial_test.rb

```diff
@@ -160,9 +160,11 @@ def test_crd_pruning
     assert_deploy_success(deploy_fixtures("crd", subset: %w(configmap-data.yml configmap2.yml)))
 
     assert_predicate build_kubectl.run("get", "mail.stable.example.io", "my-first-mail").last, :success?
-    refute_logs_match(/The following resources were pruned: mail(.stable.example.io)? "my-first-mail"/)
+    mail_separator = '[ \/]"?my-first-mail"?'
+    widget_separator = '[ \/]"?my-first-widget"?'
+    refute_logs_match(/The following resources were pruned: mail(.stable.example.io)?#{mail_separator}/)
     assert_logs_match_all([
-      /The following resources were pruned: widget(.stable.example.io)? "my-first-widget"/,
+      /The following resources were pruned: widget(.stable.example.io)?#{widget_separator}/,
```

Comment:
with k8s ci the res is printed as `widget.stable.example.io/my-first-widget` so I switched the regex but policial was yelling at me to use `%r{}` when I tried to use `[ \/]` but I couldn't figure it out so ended up using a variable for the separator section. If anyone is better at regex and knows how to please policial let me know

Reply:
`%r{The following resources were pruned: widget(.stable.example.io)?/"?my-first-widget"?}` didn't do what you wanted?